README.md

```diff
@@ -4,6 +4,7 @@
 1. minimap2 (https://github.com/lh3/minimap2) (must be available via commandline)
 2. blast tools (ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
 3. R and RSCRIPT (https://www.r-project.org/)
```

Comment:
We can remove this dependency, right?

Reply:
True!

Reply:
Current code still tries to support both HIVSeqinR and HIVIntact.
So unless we drop HIVSeqinR, we need the dependency on R.